src/worldstate/README.md

```diff
@@ -0,0 +1,7 @@
+Purpose:
```

Comment:
Nitpick, but this would read a bit better (in rendered markdown) if we replaced it with `## Purpose` (which makes it a "medium" header)

Reply:
Added